packages/components/src/components/button/button.scss

```diff
@@ -225,6 +225,9 @@
             margin-left: 8px;
         }
     }
+    &__token-create {
```

Comment:
Can we use more global naming? like button-group?

Reply:
Yes, changed relevant styles.